mkdocs.yml

```diff
@@ -29,12 +29,16 @@ nav:
       - FrameIterator: api/iterators/frame-iterator.md
     - DataFrame: api/data-frame.md
     - Table: api/table.md
-  - Pixeltable Functions:
-    - Huggingface: api/functions/huggingface.md
+  - Built-in UDFs:
     - image: api/functions/image.md
     - string: api/functions/string.md
     - timestamp: api/functions/timestamp.md
     - video: api/functions/video.md
+    - fireworks: api/functions/fireworks.md
```

Comment:
I feel like these are thematically different than the functions for our supported data types.